content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -115,6 +121,10 @@ You can also install kops using [Homebrew](https://docs.brew.sh/Homebrew-on-Linu
 brew update && brew install kops
 ```
 
+{{% /tab %}}
```

Comment:
I see (lines 120 -122) brew update and brew install listed for both tabs. Is this correct?

Reply:
line 187, I_think_ the command should be in single quotes:
Run `kops create cluster` to create your cluster configuration:

Reply:
nit: line 226, remove ampersand and replace with:
... visualization, and control of your Kubernetes cluster.

Reply:
nit:
What do you think about removing the `Feedback` section contents to the `What's Next` section?
You could add a list item, such as `Contribute to kops`
Every docs page has a `Feeback` section as the last section on the page.

Reply:
Am ok with removing the `Feedback` section, see the moved content and the re-wording.

Reply:
👍